gtfs2ntfs/src/main.rs

```diff
@@ -58,6 +58,19 @@ struct Opt {
         default_value = &transit_model::CURRENT_DATETIME
     )]
     current_datetime: DateTime<FixedOffset>,
+
+    // The max distance in meters to compute the tranfer
+    #[structopt(long, short = "d", default_value = "500")]
```

Comment:
It will be easier with constants in lib.rs (like & transit_model::CURRENT_DATETIME) just above. 
We can use them in TT too.

Reply:
done